tests/common/matchers.py

```diff
@@ -1,30 +0,0 @@
-"""
-Objects that match other objects for testing.
-
-Implements some matching objects in the style of h_matchers library for
-comparing with other objects in tests.
-"""
-
-from h_matchers.matcher.core import Matcher
-from pyramid.httpexceptions import HTTPFound, HTTPSeeOther
-
-
-class Redirect302To(Matcher):
-    """Matches any HTTPFound redirect to the given URL."""
-
-    def __init__(self, location):
-        super().__init__(
-            f"* any redirect to: {location} *",
-            lambda other: isinstance(other, HTTPFound) and other.location == location,
-        )
-
-
-class Redirect303To(Matcher):
-    """Matches any HTTPSeeOther redirect to the given URL."""
-
-    def __init__(self, location):
-        super().__init__(
-            f"* any redirect to: {location} *",
-            lambda other: isinstance(other, HTTPSeeOther)
-            and other.location == location,
-        )
```

Comment:
This was unused. Coverage should have prevented this from being possible but it seems coverage is [only being applied to `tests/unit/`](https://github.com/hypothesis/h/blob/a578983be382a4ce50c2b0e942f9cb3173046e55/pyproject.toml#L86) rather than to the entire `tests/` directory. We should fix that so that unused factories, matchers, fixtures, and other test helpers are found (`tests/functional/` will still have to be excluded from coverage unless we add coverage collection to functests).